src/utils/theme/index.js

```diff
@@ -40,9 +40,11 @@ export const COLORS = {
 
   // new color
   signed_primary: '#4782D7',
+  signed_primary_15: '#4782D726',
   signed_secondary: '#6295DD',
   default_signed_secondary: '#69A2F2',
   anon_primary: '#107793',
+  anon_primary_15: '#10779326',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color codes `signed_primary_15` and `anon_primary_15` are not in the correct format. The hexadecimal color code should be 6 characters long, but here it is 8 characters long. Please check and correct these color codes.
```diff
-   signed_primary_15: '#4782D726',
-   anon_primary_15: '#10779326',
+   signed_primary_15: '#4782D7',
+   anon_primary_15: '#107793',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->